workers/data_refinery_workers/processors/smashing_utils.py

```diff
@@ -304,6 +310,10 @@ def process_frames_for_key(key: str,
                                computed_file=computed_file.id,
                                dataset_id=job_context['dataset'].id,
                                job_id=job_context["job"].id)
+                job_context['filtered_samples'][sample.accession_code] = {
+                    'reason': 'We were unable to smash the file associated with this sample during the first pass.',
```

Comment:
This reason is going to go into the metadata file that we give our users. They won't know what "first pass" and "second pass" mean and we should avoid the use of the word "smash". A better reason to me would be: `The file associated with this sample did not pass the QC checks we apply before aggregating.` or something.